tests/Tests/800__culler/culler.robot

```diff
@@ -70,10 +70,17 @@ Verify Culler Does Not Kill Active Server
     [Tags]    Tier2
     ...       ODS-1253
     ...       Execution-Time-Over-15m
+    ...       AutomationBug
```

Comment:
Yeah I agree (read comments by Jorge's from https://github.com/red-hat-data-services/ods-ci/pull/539#discussion_r961443902)

Maybe from line 74 to 83 can be gathered into "Spawn Jupyter Notebook Active For X Minutes"
And then there can be "Wait For Culler timeout" and "Check Notebook Pod is still up" (or similar)

basides, should this TC have a teardown to close the notebook afterwards?

Reply:
it would be more for TC readability rather than reusability

Reply:
I don't think it's that hard to understand the TC, it's only running a nb and waiting for a timeout, but ok